hiddenEdit.php

```diff
@@ -1,11 +1,16 @@
 <?php
 
-session_start();
 require_once 'functions.php';
 
 $id = targetID();
 $db = getDb();
 $editSeason = filter_var($_POST['editSeason'], FILTER_VALIDATE_INT);
+if (filter_var($_POST['editSeason'], FILTER_VALIDATE_INT)) {
+    echo $editSeason;
```

Comment:
if it is valid, do you want to echo it? surely you just want to make the edit

Reply:
echo removed